src/FESpaces/FEOperators.jl

```diff
@@ -22,13 +22,26 @@ function allocate_residual(op::FEOperator,u)
   @abstractmethod
 end
 
+function doc_inplace_version(f)
+    # TODO move this helper elsewhere
+    "Inplace version of [`$f`](@ref)."
+end
```

Comment:
Suggestions where this helper function could live? Or is it over-engineered for it to exist?

Reply:
I feel it is a little bit over-engineering.
I would write it explicitly every time. It is about the same length.